src/app/modules/create-user/services/create-user.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { CreateUser } from 'src/app/common/interfaces/createUser';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
din nou nu trebuie sa fie provided in root daca tu il folosesti doar in modulul asta